### PR TITLE
Remove industry code from VBA request in favor of random

### PR DESCRIPTION
**Pull request by Dal-Papa**
@Expensify/pullerbear : 🐻 

Hold for https://github.com/Expensify/Web-Secure/pull/2026

### Details
The industry classification code field on the Company Information step of bank account connection in NewDot is a sticking point for users. Many are unaware of what it is or that it needs to be six digits. Remove the field from the NewDot VBA form and automatically assign a valid 6 digit industry classification code.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/176430

### Tests & QA

1. Navigate to `/bank-account/company`
2. Validate that you don't see the "Industry classification code" field

### Tested On

- [x] Web
- [x] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web

Screen Shot 2021-09-07 at 19 13 13

#### Mobile Web

Screen Shot 2021-09-07 at 19 15 43

### CODE REVIEW

**Review by joelbettner — COMMENTED** [review]
> Remove the field from the NewDot VBA form and automatically assign a valid 6 digit industry classification code.

I see that we are removing it, but I don't see where we are automatically assigning a valid 6 digit industry classification code. Is that coming in a separate PR?

**Comment by joelbettner** [comment]
@Dal-Papa bump^.

**Comment by luacmartins** [comment]
Stepping in here. I believe the new code is being assigned as part of this PR https://github.com/Expensify/Web-Secure/pull/2026/files

File: src/pages/ReimbursementAccount/CompanyStep.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ import TextLink from '../../components/TextLink';
 import StatePicker from '../../components/StatePicker';
 import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 import {
-    isValidAddress, isValidDate, isValidIndustryCode, isValidZipCode,
```
Comment (luacmartins) [inline src/pages/ReimbursementAccount/CompanyStep.js:28]: NAB we could also remove `isValidIndustryCode` now that it's no longer used.

**Review by luacmartins — DISMISSED** [review]
Code LGTM and tests well. Just gotta take care of conflicts now.

**Comment by luacmartins** [comment]
Coming from here https://expensify.slack.com/archives/C020EPP9B9A/p1631889690477900, I'm stepping in to get this merged ASAP.

**Review by timszot — APPROVED** [review]
LGTM! Approving, feel free to merge once Travis is happy.

**Pull request merged**